packages/trader/src/Modules/Trading/Helpers/__tests__/contract-type.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+    getContractTypeCategoryIcons,
+    getAvailableContractTypes,
+    getContractCategoryKey,
+    getContractTypes,
+} from '../contract-type';
+
+const contract_types_test_list = {
+    Accumulators: { name: 'Accumulators', categories: [{ value: 'accumulator', text: 'Accumulator' }] },
+    Digits: { name: 'Digits', categories: [{ value: 'match_diff', text: 'Matches/Differs' }] },
+    'Ins & Outs': { name: 'Ins & Outs', categories: [{ value: 'end', text: 'Ends In/Ends Out' }] },
+};
+const unsupported_test_list = ['end', 'stay'];
+const unsupported_short_test_list = ['stay'];
+const contract_type_array = [
+    { value: 'accumulator', text: 'Accumulator' },
+    { value: 'rise_fall', text: 'Rise/Fall' },
+];
+const contract_categoty_list = [
+    {
+        contract_types: contract_type_array,
+        contract_categories: [
+            {
+                contract_types: contract_type_array,
+                icon: 'IcCatAll',
+                key: 'All',
+                label: 'All',
+            },
+        ],
+        icon: 'IcCatAll',
+        key: 'All',
+        label: 'All',
+    },
+    {
+        contract_types: [{ value: 'Multipliers', text: 'Multiplierss' }],
+        contract_categories: [
+            {
+                contract_types: [{ value: 'multipliers', text: 'Multipliers' }],
+                icon: 'IcCatMultiplier',
+                key: 'Multipliers',
+                label: 'Multipliers',
+            },
+        ],
+        icon: 'IcCatMultiplier',
+        key: 'Multipliers',
+        label: 'Multipliers',
+    },
+];
+
+describe('getContractTypeCategoryIcons', () => {
```

Comment:
In `contract-type.tsx` there were 4 functions in one file to be tested (`getContractTypeCategoryIcons, getAvailableContractTypes, getContractCategoryKey, getContractTypes`), that is why I created a separate `describe` section for each of them.